launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -100,31 +114,49 @@ def __exit__(self, exep_type, exep_value, trace):
 class _WaitForTopicsNode(Node):
     """Internal node used for subscribing to a set of topics."""
 
-    def __init__(self, name='test_node', node_context=None):
-        super().__init__(node_name=name, context=node_context)
+    def __init__(
+            self, name="test_node", node_context=None, max_number_of_messages=None
```

Comment:
Could you change the argument name, ``max_number_of_messages`` to something more specific, like ``messages_deque_length`` or ``messages_buffer_length`` ? Even the same ``messages_received_buffer_length`` doesn't seem bad.

Reply:
+1. Let's go with `messages_received_buffer_length`

Reply:
Fixed in commit 6228855463e7197cf1cb285ca8428af8f224673b